### PR TITLE
feat: [OeX_Cred-206] add program certificates list endpoint

### PR DESCRIPTION
**Pull request by Thposbsmt**
**Run JavaScript tests locally with Karma**

There is work being done on a fix to get Karma to run in CI. Until then, however, contributors are required to run these tests locally.

- [x] Make sure you are inside the devstack container
- [x] Run `make test-karma`
- [x] All tests pass

### CODE REVIEW

File: credentials/apps/verifiable_credentials/rest_api/v1/serializers.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class ProgramCredentialSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```
Comment (wowkalucky) [inline credentials/apps/verifiable_credentials/rest_api/v1/serializers.py:4]: Seems, we use this serializer for validation only?
If so, making it readonly should increase productivity.

**Review by wowkalucky — CHANGES_REQUESTED** [review]
Please, see items.

File: credentials/apps/verifiable_credentials/rest_api/v1/views.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from .serializers import ProgramCredentialSerializer
+
+
+User = get_user_model()
+
+
+class ProgramCredentialsViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+
+    # FIXME: for some reasons authentication not working
+    authentication_classes = (
```
[the remark this answers is not in the document]
Comment (Thposbsmt) [reply credentials/apps/verifiable_credentials/rest_api/v1/views.py:19]: In defaults it's `credentials.apps.api.authentication.JwtAuthentication`, we use `from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication`

**Review by wowkalucky — APPROVED** [review]
Well done!

**Pull request merged**